aptos-move/framework/aptos-stdlib/sources/signature.move

```diff
@@ -1,12 +1,37 @@
 /// Contains functions for [ed25519](https://en.wikipedia.org/wiki/EdDSA) digital signatures and for
 /// [Boneh-Lynn-Shacham (BLS) signatures](https://en.wikipedia.org/wiki/BLS_digital_signature)
 module aptos_std::signature {
-    /// Return `true` if the bytes in `public_key` is a valid bls12381 public key and
-    /// passed the PoP check.
+    use std::option::Option;
+
+    /// Given a vector of serialized public keys, combines them into an aggregated public key which can be used to verify
+    /// multisignatures using `bls12381_verify_signature`.
+    /// Returns 'None' if no public keys are given as input. This function assumes that the caller validated all input
+    /// public keys using `bls12381_validate_pubkey`.
+    /// Does not abort.
+    native public fun bls12381_aggregate_unvalidated_pubkeys(
+        public_keys: vector<vector<u8>>,
+    ): Option<vector<u8>>;
+
+    // TODO: implement remaining BLS functions:
+    // native public fun bls12381_aggregate_signatures(signatures: vector<vector<u8>>): Option<vector<u8>>;
+    // native public fun bls12381_verify_aggregate_signature(
+    //        signature: vector<u8>,
+    //        public_keys: vector<vector<u8>>,
+    //        messages: vector<vector<u8>>,
+    //    ): bool;
+
+    /// Return `true` if the bytes in `public_key` are a valid bls12381 public key: it is in the prime-order subgroup
+    /// and it is different from the identity group element.
     /// Return `false` otherwise.
     /// Does not abort.
-    native public fun bls12381_validate_pubkey(public_key: vector<u8>, proof_of_possesion: vector<u8>): bool;
```

Comment:
I chose to rename this to `bls1381_verify_proof_of_possesion` and keep `bls12381_validate_pubkey` for doing the prime-order subgroup check. This is consistent with our terminology for Ed25519 signatures in this Move file, which also use `ed25519_valida_pubkey` to make sure a public key is not a small-order point (although it could still be composite-order, not necessarily prime-order).

It's also better to name the PoP verification `bls1381_verify_proof_of_possesion` because we might soon need to add a similarly-named `ed25519_verify_proof_of_knowledge` API for a proof-of-knowledge (PoK) verification in Ed25519. (A PoP and a PoK basically help us achieve the same thing; the devil is in the details of the security proof.)